spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -289,6 +291,7 @@ def main(argv=None):
         print_ok()
     except ImportError:
         print_fail()
+        print("Unable to import spinalcordtoolbox module!?")
```

Comment:
Nitpicky suggestions: 

* You can add the message directly to `print_fail()`
* As much as I love interrobangs, I think a period might strike a better tone? (Super nitpicky, sorry!)

```suggestion
        print_fail("Unable to import spinalcordtoolbox module.")
```